### PR TITLE
Update DSSP tests to check DSSP version before running mmCIF tests

### LINKED ISSUE #1200: DSSP failing on new mmCIF tests
According to the buildbot http://testing.open-bio.org/biopython/tgrid a recent change on the master has broken DSSP testing, 7a42665d5fce fails while a172cb930029 passed.

e.g. http://testing.open-bio.org/biopython/builders/Linux%2064%20-%20Python%203.4/builds/832/steps/shell/logs/stdio

```
======================================================================
ERROR: test_dssp_with_mmcif_file (test_DSSP_tool.DSSP_test)
Test DSSP generationg from MMCIF
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home_local/buildslave/BuildBot_Biopython/lin3464/build/Tests/test_DSSP_tool.py", line 53, in test_dssp_with_mmcif_file
    dssp = DSSP(model, pdbfile)
  File "/home_local/buildslave/BuildBot_Biopython/lin3464/build/build/lib.linux-x86_64-3.4/Bio/PDB/DSSP.py", line 356, in __init__
    dssp_dict, dssp_keys = dssp_dict_from_pdb_file(in_file, dssp)
  File "/home_local/buildslave/BuildBot_Biopython/lin3464/build/build/lib.linux-x86_64-3.4/Bio/PDB/DSSP.py", line 196, in dssp_dict_from_pdb_file
    raise Exception('DSSP failed to produce an output')
Exception: DSSP failed to produce an output

======================================================================
ERROR: test_dssp_with_mmcif_file_and_nonstandard_residues (test_DSSP_tool.DSSP_test)
Test DSSP generationg from MMCIF
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home_local/buildslave/BuildBot_Biopython/lin3464/build/Tests/test_DSSP_tool.py", line 61, in test_dssp_with_mmcif_file_and_nonstandard_residues
    dssp = DSSP(model, pdbfile)
  File "/home_local/buildslave/BuildBot_Biopython/lin3464/build/build/lib.linux-x86_64-3.4/Bio/PDB/DSSP.py", line 356, in __init__
    dssp_dict, dssp_keys = dssp_dict_from_pdb_file(in_file, dssp)
  File "/home_local/buildslave/BuildBot_Biopython/lin3464/build/build/lib.linux-x86_64-3.4/Bio/PDB/DSSP.py", line 196, in dssp_dict_from_pdb_file
    raise Exception('DSSP failed to produce an output')
Exception: DSSP failed to produce an output

----------------------------------------------------------------------
```

Given the timing and failure 67bc849e77688e4be13f69e8477920122389d64d seems likely the cause from #1194, CC @andrewguy 

Might this test machine have too old a version of DSSP?

### PR DESCRIPTION
**Pull request by andrewguy**
Pull request to fix #1200

### CODE REVIEW

**Comment by peterjc** [comment]
Thanks! That looks sensible - I'll have to try it on the machine with the old DSSP installed first though...

File: Tests/test_DSSP_tool.py [hunk 1]
```diff
@@ -46,20 +54,23 @@ def test_dssp(self):
...
     def test_dssp_with_mmcif_file(self):
-        """Test DSSP generationg from MMCIF"""
-        p = MMCIFParser()
-        pdbfile = "PDB/2BEG.cif"
-        model = p.get_structure("2BEG", pdbfile)[0]
-        dssp = DSSP(model, pdbfile)
-        self.assertEqual(len(dssp), 130)
+        """Test DSSP generation from MMCIF"""
+        # Only run tests if DSSP version installed supports mmcif
+        if StrictVersion(dssp_version) >= StrictVersion('2.2.0'):
```
Comment (peterjc) [inline Tests/test_DSSP_tool.py]: Does it make sense to put this if statement outside the ``def``? i.e. Only define this test (and the one following) if using a recent DSSP?

Comment (andrewguy) [inline Tests/test_DSSP_tool.py]: Yep, good catch. Updated accordingly. Thanks!

**Comment by peterjc** [comment]
Thank you, closing #1200

**Pull request merged**